runtime/onert/core/include/ir/train/operation/DepthwiseConv2D.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_TRAIN_OPERATION_DEPTHWISECONV2D_H__
+#define __ONERT_IR_TRAIN_OPERATION_DEPTHWISECONV2D_H__
```

Comment:
```suggestion
#ifndef __ONERT_IR_TRAIN_OPERATION_DEPTHWISE_CONV2D_H__
#define __ONERT_IR_TRAIN_OPERATION_DEPTHWISE_CONV2D_H__
```
This is more readable, and I've checked `TrainableOperationConverter.h` added `_` for Camel file name between words.

Reply:
This code came from https://github.com/Samsung/ONE/blob/c6b76d112b9c902055457611359268b27c5c9e07/runtime/onert/core/include/ir/operation/DepthwiseConv2D.h#L17 file. Anyway, I updated this PR.